wagtail/wagtailcore/tests/test_page_model.py

```diff
@@ -67,13 +67,6 @@ def test_slug_is_autogenerated(self):
         retrieved_page = Page.objects.get(id=events_page.id)
         self.assertEqual(retrieved_page.slug, 'events-2')
 
-    def test_slug_must_be_unique_within_parent(self):
```

Comment:
If we're changing the behaviour from "a duplicate slug produces a ValidationError" to "a duplicate slug is silently changed into a non-duplicate one" (and, to be honest, I'm not sure that we _should_ do that...), then we need a new test to confirm that the page really is saved successfully.